features/wishlist.feature

```diff
@@ -135,3 +135,24 @@ Scenario: Get a Wishlist Item
     And I set the "Product ID - Search Product" to "1024"
     And I press the "Search - Product" button
     Then I should see Product ID = "1024", Product Name = "Powers of 2" in the results
+
+Scenario: Update Wishlist Product
+    Given The service is running
+    When I visit the "home page"
+    And I set the "Wishlist Name - Create" to "WISHLIST"
+    And I set the "Customer ID - Create" to "10"
+    And I press the "Create" button
+    Then I should see the message "Success"
+    When I copy the "Wishlist ID - Create" field
+    And I press the "Clear" button
+    And I press the "Wishlist Item" tab
+    When I paste the "Wishlist ID - Create Product" field
+    And I set the "Product ID - Create Product" to "1024"
+    And I set the "Product Name - Create Product" to "Powers of 2"
+    And I press the "Create - Product" button
+    Then I should see the message "Success"
+    When I paste the "Wishlist ID - Update Product" field
+    And I set the "Product ID - Update Product" to "1024"
+    And I set the "Product Name - Update Product" to "Item 1024"
+    And I press the "Update - Product" button
+    Then I should see the message "Success"
```

Comment:
Could you search and check if the new name exists in the result list?

Reply:
done